src/components/character-count/index.md.njk

```diff
@@ -45,8 +45,13 @@ This component uses JavaScript. If JavaScript is not available, users will see a
 
 There are 2 ways to use the character count component. You can use HTML or, if you’re using Nunjucks or the GOV.UK Prototype Kit, you can use the Nunjucks macro.
 
-{% from "_example.njk" import example %}
-{{ example({group: "components", item: "character-count", example: "label-page-heading", html: true, nunjucks: true, open: false}) }}
+{{ example({group: "components", item: "character-count", example: "default", html: true, nunjucks: true, open: false}) }}
+
+###  If you’re asking more than one question on the page
+
+If you’re asking more than one question on the page, do not set the contents of the `<label>` as the page heading.
```

Comment:
In instances of 'If you’re asking more than one question on the page', I'd suggest linking to the Questions pages page, so a user understands when to do it, and how to style the <label> and <legends>.

Add to end of sentence:
```
[See Question pages](../../patterns/question-pages/#asking-multiple-questions-on-a-page)
```